hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsStatistics.java

```diff
@@ -32,10 +32,54 @@
 public class ITestAbfsStatistics extends AbstractAbfsIntegrationTest {
 
   private static final int NUMBER_OF_OPS = 10;
+  private String createOp =
```

Comment:
Why you created local variables? Just use directly wherever required. These are just string constants.

Reply:
Since I was using these values, again and again, I thought to localize them and also increase the readability of the code. This is was just simply for better readability and have fewer lines of code.